app/helper_test.go

```diff
@@ -499,22 +500,14 @@ func (me *TestHelper) ResetEmojisMigration() {
 
 func (me *TestHelper) CheckTeamCount(t *testing.T, expected int64) {
 	teamCount, err := me.App.Srv.Store.Team().AnalyticsTeamCount()
-	if err != nil {
-		t.Fatalf("Failed to get team count.")
-	}
-	if teamCount != expected {
-		t.Fatalf("Unexpected number of teams. Expected: %v, found: %v", expected, teamCount)
-	}
+	require.Nil(t, err, "Failed to get team count.")
+	require.Equal(t, teamCount, expected, "Unexpected number of teams. Expected: %v, found: %v", expected, teamCount)
```

Comment:
nit: I would use `Equalf` for error customization.
see https://github.com/stretchr/testify/pull/356 for the details.